doc/UserGuide.md

```diff
@@ -193,6 +193,15 @@ see the Mserialize documentation on [Adapting custom types][mserialize-act] and
 [mserialize-act]: Mserialize.html#adapting-custom-types
 [mserialize-rec]: Mserialize.html#adapting-user-defined-recursive-types-for-visitation
 
+## Logging Standard Types
+
+`std::chrono::system_clock::time_point` objects can be logged after adoption by including a header file:
+
+    [catchfile test/integration/LoggingTimePoint.cpp timepoint]
+
+The object is serialized as a single number (nanoseconds since epoch), and pretty printed as a human readable timestamp.
+The pretty printed text representation is in UTC, and can be configured with the `-d` flag when using [bread](#bread).
```

Comment:
https://github.com/morganstanley/binlog/blob/main/bin/bread.cpp#L8 default timestamps on log lines still looks to be in local time, whereas this comment says UTC for timestamps in the log data. I recommend we unify on the same handling for both log line timestamps and timestamps within the user log data. I'm biased but I'd vote for a default of UTC across the board.

Reply:
I added a test that verifies this. Why to you think it is using localtime?

Reply:
https://github.com/morganstanley/binlog/blob/main/bin/bread.cpp#L8 BINLOG_DEFAULT_FORMAT uses %d instead of %u for the timestamps

Reply:
Oh, I see what you mean now.

Reply:
Ok, this is definitely confusing. I'm vary of changing defaults though. What do you think about this solution: if the specified format contains "%d" before "%u", inline timestamps are rendered in producer local time (if possible), otherwise they are rendered in UTC:

-f "%d": localtime everywhere
-f "%u": utc everywhere
-f "%u %d" utc then localtime in the event timestamp, utc in the inline timestamp
-f "%d %u" localtime then utctime in the event timestamp, localtime in the inline timestamp

Reply:
Sounds good

Reply:
Implemented, tested, documented.